ionic2/app/components/filter-popover/filter-popover.component.ts

```diff
@@ -1,13 +1,30 @@
 import { Component } from '@angular/core';
-import { Events, ViewController } from 'ionic-angular';
+import { Events, ViewController, NavParams } from 'ionic-angular';
 import { Filter } from '../../models/filter';
 import { ApiService } from '../../services/api.service';
 import { ConfigService } from '../../services/config.service';
 import { PokeFilterTimeTabComponent } from '../poke-filter-time-tab/poke-filter-time-tab.component';
 import { PokeFilterPokemonTabComponent } from '../poke-filter-pokemon-tab/poke-filter-pokemon-tab.component';
 
 @Component({
-  template: require('./filter-popover.component.html'),
+  template: `
```

Comment:
Uhm ... I see xD

If this really fixes it or will break it if this inline template is outsourced, there should be a comment stating that this is **intentional** and **important** :)